soccer/planning/planner/EscapeObstaclesPathPlanner.cpp

```diff
@@ -31,12 +32,17 @@ Trajectory EscapeObstaclesPathPlanner::plan(const PlanRequest& planRequest) {
     Geometry2d::ShapeSet obstacles;
     FillObstacles(planRequest, &obstacles, nullptr, false, nullptr);
 
-    if (!obstacles.hit(startInstant.position())) {
-        // Stop in place
+    // TODO(#
```

Comment:
lost the todo number

Reply:
Yeah, oops. This TODO actually shouldn't exist, it's fine for now.